sources/packages/backend/apps/db-migrations/src/sql/Reports/Update-data-inventory-report.sql

```diff
@@ -0,0 +1,527 @@
+				and dv.value_code = ''CSLF''
+				group by a.id
+			) as award_disbursements_cslf on award_disbursements_cslf.application_id = applications.id
+			left join (
+				select a.id as application_id, sum(dv.effective_amount) as effective_amount
+				from sims.disbursement_schedules ds, sims.disbursement_values dv, sims.applications a, sims.student_assessments sa
+				where dv.disbursement_schedule_id = ds.id
+				and ds.student_assessment_id = sa.id
+				and sa.application_id = a.id
+				and dv.value_code = ''CSLP''
+				group by a.id
+			) as award_disbursements_cslp on award_disbursements_cslp.application_id = applications.id
+			left join (
+				select a.id as application_id, sum(dv.effective_amount) as effective_amount
+				from sims.disbursement_schedules ds, sims.disbursement_values dv, sims.applications a, sims.student_assessments sa
+				where dv.disbursement_schedule_id = ds.id
+				and ds.student_assessment_id = sa.id
+				and sa.application_id = a.id
+				and dv.value_code = ''BCSL''
+				group by a.id
+			) as award_disbursements_bcsl on award_disbursements_bcsl.application_id = applications.id
+			left join (
+				select a.id as application_id, sum(dv.effective_amount) as effective_amount
+				from sims.disbursement_schedules ds, sims.disbursement_values dv, sims.applications a, sims.student_assessments sa
+				where dv.disbursement_schedule_id = ds.id
+				and ds.student_assessment_id = sa.id
+				and sa.application_id = a.id
+				and dv.value_code = ''CSGD''
+				group by a.id
+			) as award_disbursements_csgd on award_disbursements_csgd.application_id = applications.id
+			left join (
+				select a.id as application_id, sum(dv.effective_amount) as effective_amount
+				from sims.disbursement_schedules ds, sims.disbursement_values dv, sims.applications a, sims.student_assessments sa
+				where dv.disbursement_schedule_id = ds.id
+				and ds.student_assessment_id = sa.id
+				and sa.application_id = a.id
+				and dv.value_code = ''CSGP''
+				group by a.id
+			) as award_disbursements_csgp on award_disbursements_csgp.application_id = applications.id
+			left join (
+				select a.id as application_id, sum(dv.effective_amount) as effective_amount
+				from sims.disbursement_schedules ds, sims.disbursement_values dv, sims.applications a, sims.student_assessments sa
+				where dv.disbursement_schedule_id = ds.id
+				and ds.student_assessment_id = sa.id
+				and sa.application_id = a.id
+				and dv.value_code = ''CSGF''
+				group by a.id
+			) as award_disbursements_csgf on award_disbursements_csgf.application_id = applications.id
+			left join (
+				select a.id as application_id, sum(dv.effective_amount) as effective_amount
+				from sims.disbursement_schedules ds, sims.disbursement_values dv, sims.applications a, sims.student_assessments sa
+				where dv.disbursement_schedule_id = ds.id
+				and ds.student_assessment_id = sa.id
+				and sa.application_id = a.id
+				and dv.value_code = ''CSGT''
+				group by a.id
```

Comment:
Was there a reason to use the join clauses as where conditions for these queries?
I would prefer to keep the same style from the other queries, unless I am missing something here.

Reply:
No, there's no especial reason. Agree with you about the style. I just couldn't work on that.